shapely/constructive.py

```diff
@@ -436,6 +437,40 @@ def delaunay_triangles(geometry, tolerance=0.0, only_edges=False, **kwargs):
     return lib.delaunay_triangles(geometry, tolerance, only_edges, **kwargs)
 
 
+@requires_geos("3.10.0")
+@multithreading_enabled
+def constrained_delaunay_triangles(geometry, **kwargs):
+    """Computes a constrained Delaunay triangulation around the vertices of an input
+    Polygon or MultiPolygon.
+
+    The output is a GeometryCollection of Polygons.
+
+    For non-polygonal inputs, returns an empty geometry collection.
+
+    Parameters
+    ----------
+    geometry : Geometry or array_like
+    **kwargs
+        For other keyword-only arguments, see the
+        `NumPy ufunc docs <https://numpy.org/doc/stable/reference/ufuncs.html#ufuncs-kwargs>`_.
+
```

Comment:
Can you add a "Returns" section? 

And I would also add a "See Also" section (see eg how this is used in `reverse` in this file) to point to `delaunay_triangles`, and also add this to the docstring of `delaunay_triangles` to point to `constrained_delaunay_triangles`